pkg/proxy/ipvs/proxier.go

```diff
@@ -309,7 +309,11 @@ func (r *realIPGetter) NodeIPs() (ips []net.IP, err error) {
 	}
 	// translate ip string to IP
 	for _, ipStr := range nodeAddress.UnsortedList() {
-		ips = append(ips, net.ParseIP(ipStr))
+		a := net.ParseIP(ipStr)
+		if a.IsLoopback() {
+			continue
+		}
+		ips = append(ips, a)
```

Comment:
we should also modify the comment of this func correspondingly,.
https://github.com/kubernetes/kubernetes/blob/8edda782f30a377afd031d5d7f4d62551785c441/pkg/proxy/ipvs/proxier.go#L303

Reply:
Comment updated.